### DISCUSSION TITLE
function extensions.http_post(unknown, text, http_header!|) does not exist

### DISCUSSION THREAD

**Discussion by seekinmonky** [supabase/supabase#22085] [Questions] [answered]
In my users table, when is_verified column is changed to true, i want to send an email to the user saying "your account" is verified,
for sending the email i set up a edge function (which is working fine) and a trigger. the trigger will call a database function, which will make a post request to the edge function, but somehow, my database function is showing the folowing error : function extensions.http_post(unknown, text, http_header!|) does not exist

attaching code of the trigger and database function
`CREATE TRIGGER on_user_verification_update
AFTER UPDATE ON public.users
FOR EACH ROW
WHEN (OLD.is_verified IS DISTINCT FROM NEW.is_verified)
EXECUTE PROCEDURE public.send_verification_email();
`

____

CREATE OR REPLACE FUNCTION public.send_verification_email()
RETURNS TRIGGER AS $$
DECLARE
    response json;
    jsonData jsonb;
BEGIN
    IF (NEW.is_verified = true AND OLD.is_verified = false) THEN
        jsonData := jsonb_build_object(
            'name', NEW.name,
            'email', NEW.email
        );
        
        jsonData := jsonData::text;
        
        SELECT extensions.http_post(
            'https://dummyurl.supabase.co/functions/v1/sendVerificationConfirmation', 
            jsonData, 
            'application/json' 
        ) INTO response;
    END IF;
    RETURN NEW; 
END;
$$ LANGUAGE plpgsql SECURITY DEFINER;

pls help

**Comment by GaryAustin1** [comment, marked as answer]
Assuming you have the extension enabled...
Then you are not using the correct format for a jsonb data post.  If you do jsonb instead of varchar you use the 2nd form of the call.

![image](https://github.com/supabase/supabase/assets/54564956/16c4e9d9-5ec6-4d31-94d6-20f981f131a3)

**Comment by seekinmonky** [comment]
Thanks @GaryAustin1 for your help
